### PR TITLE
Product detail

### PR DESCRIPTION
**Pull request by Ranjeetk489**
I'm sorry for the pull request showing 17 files.. for this Pull request i need just 2 files to be checked named product-detail.html and product-detail-css and rest can be ignored as they were reviewed in previous PRs... i'll take care of this mistake from next PR.. Thanks

### CODE REVIEW

File: components/UserManagement/main.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  justify-content: center;
+  align-items: center;
+  padding: 2rem 3rem;
+  background-color: var(--white-color);
+  margin: 0 auto;
+  margin-top: 8vh;
+  width: fit-content;
+}
+.form-ctrl {
+  margin-bottom: 0.8rem;
```
Comment (bossmonkey98) [inline components/UserManagement/main.css]: use px for values less than 10px

File: components/checkout/checkout.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            </span>
+            <div class="nav-links">
+                <a href="" class="nav-link">Watches</a>
+                <a href="" class="nav-link">Eyewear</a>
+                <a href="" class="nav-link">Accessories</a>
+                <a href="" class="nav-link">News</a>
+            </div>
+            <div class="sec-nav-links nav-links">
+                <a href="" class="search-icon nav-link">
+                    <img src="../../assets/search.svg" alt="" class="search-icon" />
```
Comment (bossmonkey98) [inline components/checkout/checkout.html:33]: add text input section beside search icon

**Comment by bossmonkey98** [comment]
Add bottom border / shadow to navbar

**Review by bossmonkey98 — APPROVED** [review]

**Comment by bossmonkey98** [comment]
the background of the bag icon looks kinda oval make it circle

**Review by rutu14 — APPROVED** [review]
In the testimonials part, put the text aside completely or make the background for text over image different. It is not readable over the image.

**Comment by Ranjeetk489** [comment]
> In the testimonials part, put the text aside completely or make the background for text over image different. It is not readable over the image.

I haven't worked on site responsiveness yet if you visit the website on larger screen then you'll see the text is seperated from the testimonial part... but thanks for pointing it out i'll create a new pr to work on the responsiveness issue

**Comment by Ranjeetk489** [comment]
> the background of the bag icon looks kinda oval make it circle

okay will take a look

File: components/UserManagement/main.css (hunk 1 above)
Comment (Ranjeetk489) [reply components/UserManagement/main.css]: > Add bottom border / shadow to navbar

is it design preference? i feel it looks better without the shadow.

**Comment by Ranjeetk489** [comment]
> the background of the bag icon looks kinda oval make it circle

changed it in other pr

**Pull request merged**